server/controllers/refer/referralsController.ts

```diff
@@ -14,6 +14,50 @@ export default class ReferralsController {
     private readonly referralService: ReferralService,
   ) {}
 
+  checkAnswers(): TypedRequestHandler<Request, Response> {
+    return async (req: Request, res: Response) => {
+      TypeUtils.assertHasUser(req)
+
+      const { referralId } = req.params
+      const { username } = req.user
+
+      const referral = await this.referralService.getReferral(req.user.token, referralId)
+      const course = await this.courseService.getCourseByOffering(req.user.token, referral.offeringId)
+      const courseOffering = await this.courseService.getOffering(req.user.token, referral.offeringId)
+      const organisation = await this.organisationService.getOrganisation(req.user.token, courseOffering.organisationId)
+      const person = await this.personService.getPerson(req.user.token, referral.prisonNumber)
+
+      if (!organisation) {
+        throw createError(404, {
+          userMessage: 'Organisation not found.',
+        })
+      }
+
+      if (!person) {
+        throw createError(404, {
+          userMessage: `Person with prison number ${req.params.prisonNumber} not found.`,
+        })
+      }
+
+      const coursePresenter = CourseUtils.presentCourse(course)
+
+      res.render('referrals/checkAnswers', {
+        applicationSummaryListRows: ReferralUtils.applicationSummaryListRows(
+          courseOffering,
+          coursePresenter,
+          organisation,
+          person,
+          username,
+        ),
+        courseOfferingId: courseOffering.id,
```

Comment:
Do we need to pass `courseOfferingId` down? Where is it used?

Reply:
Good spot, was used in a previous commit but no longer needed.